src/main/scala/com/cognite/sdk/scala/v1/resources/assets.scala

```diff
@@ -50,7 +52,7 @@ class Assets[F[_]](val requestSession: RequestSession[F])
   private[sdk] def filterWithCursor(
       filter: AssetsFilter,
       cursor: Option[String],
-      limit: Option[Int],
+      limit: Option[Int] = Some(filterLimit),
```

Comment:
I don't think this is what we want, to be honest. Let's go through this a bit. Again, we shouldn't be setting a limit for how much we return from the filter here, we want to set the `limit` parameter (which again should be called pageSize in my opinion) later on. And it shouldn't be a configurable setting either, at least not initially.